exchanges/exchange.go

```diff
@@ -1752,3 +1751,48 @@ func (b *Base) MatchSymbolCheckEnabled(symbol string, a asset.Item, hasDelimiter
 func (b *Base) IsPairEnabled(pair currency.Pair, a asset.Item) (bool, error) {
 	return b.CurrencyPairs.IsPairEnabled(pair, a)
 }
+
+// Start starts an exchange bootstrap process in the background
+func Start(ctx context.Context, b IBotExchange, wg *sync.WaitGroup) error {
+	if wg == nil {
+		return fmt.Errorf("%T %w", wg, common.ErrNilPointer)
+	}
+	wg.Add(1)
+	go func() {
+		Run(ctx, b)
+		wg.Done()
+	}()
+	return nil
+}
+
+// Run performs a few Bootstrapping actions for the exchange
+func Run(ctx context.Context, b IBotExchange) {
```

Comment:
Suggestion: We could use another wait group and parallel every REST request here to make it a bit more snappier if you want.

Reply:
Good suggestion

Fixed 905d6555